### DISCUSSION TITLE
Intel Celeron CPU N2840

### DISCUSSION THREAD

**Discussion by tumbling-dice0001** [builtbybel/FlyOOBE#22] [General]
Hi, are there any issues with using Flyby11 on a laptop?  I'm using an ACER Aspire with an Intel Celeron CPU N2840.  Sorry for the dumb question, I'm an older dude and "PC" and "laptop" are different things to me, lol.

Thank you!

**Comment by ThomasACarlos** [comment]
I was able to update my 12 year old Dell Inspiron that has a Celeron processor with no problems.  This computer has the drive that was configured with MBR (Master Boot Record).  The BIOS mode is set to Legacy.  (Maybe this is a Dell thing?)  However, when I tried to do the same update to my 12 year old Dell Latitude, everything worked until I got to the first Restart.  At that point, the SSD could not be recognized.  So I am wondering if this has anything to do the Legacy vs UEFI or the drive formatting of MBR vs GPT.
